server/types/globals.go

```diff
@@ -24,7 +24,8 @@ import "fmt"
 type DoltgresTypeBaseID uint32
 
 const (
-	DoltgresTypeBaseID_Any DoltgresTypeBaseID = iota + 8192
+	DoltgresTypeBaseID_Invalid DoltgresTypeBaseID = iota
```

Comment:
We should remove this "invalid" type. Postgres doesn't have such a concept, so we'd only be deviating from how Postgres handles types, making it that much harder to fix discrepancies between our code and Postgres' behavior when we eventually have adequate testing for these kinds of things.